pkg/controller/node/metrics.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package node
+
+import (
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	NodeControllerSubsystem = "node_collector"
+	ZoneHealthStatisticKey  = "zone_health_key"
+	ZoneSizeKey             = "zone_size_key"
+)
+
+var (
+	ZoneHealth = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: NodeControllerSubsystem,
+			Name:      ZoneHealthStatisticKey,
+			Help:      "Gauge measuring percentage of healty nodes per zone.",
```

Comment:
I'd prefer that this be a raw number (e.g. number of healthy nodes per zone) rather than a percentage. This allows for broader usage of this metric, including allowing a dashboard creator to combine with the other "ZoneSize" metric for percentage graphs.